week4/taqueria.py

```diff
@@ -0,0 +1,27 @@
+entrees: dict = {
+    "Baja Taco": 4.00,
+    "Burrito": 7.50,
+    "Bowl": 8.50,
+    "Nachos": 11.00,
+    "Quesadilla": 8.50,
+    "Super Burrito": 8.50,
+    "Super Quesadilla": 9.50,
+    "Taco": 3.00,
+    "Tortilla Salad": 8.00,
+}
+
+total = 0.0
+
+while True:
+        try:
+            item = input("Item: ").strip().title()
```

Comment:
what exception is this line throwing and where?  when use `try` you need to make sure there is possibly error thrown.  otherwise do not need to use it, it makes the code more complicated.